layers/dot11.go

```diff
@@ -926,6 +926,7 @@ func (m *Dot11MgmtAssociationReq) NextLayerType() gopacket.LayerType {
 func (m *Dot11MgmtAssociationReq) DecodeFromBytes(data []byte, df gopacket.DecodeFeedback) error {
 	m.CapabilityInfo = binary.LittleEndian.Uint16(data[0:2])
 	m.ListenInterval = binary.LittleEndian.Uint16(data[2:4])
+	m.Payload = data[4:]
```

Comment:
would you mind also setting m.Contents here and in the other locations as well?

m.Contents is the bytes that make up THIS layer, m.Payload is the bytes that make up all subsequent ones.